build-data-capturing-gradle-samples/capture-ge-plugin-version/gradle-ge-plugin-version.gradle

```diff
@@ -0,0 +1,15 @@
+/**
+ * This Groovy script captures the Gradle Enterprise Maven Extension version as a custom value.
+ *
+ * <code> apply from: file('gradle-ge-plugin-version.gradle') </code>
+ */
+
+def buildScanApi = project.extensions.findByName('buildScan')
+if (!buildScanApi) {
+    return
+}
+
+URL url = buildScanApi.class.getClassLoader().getResource("com.gradle.scan.plugin.internal.meta.buildAgentVersion.txt")
+String buildAgentVersion = url.text
+buildScan.value("GE Gradle plugin version", buildAgentVersion)
```

Comment:
I do not feel super strong but I wonder if we should remove the `version` part from the custom value name. If we did so, similary for Maven, of course.

Reply:
I think for the sake of consistency, since we write `id` in "Git commit id", we should write `version` in "GE Gradle plugin version".